### PR TITLE
Added unapplyall patch and dbus method to daemon

### PR DESCRIPTION
**Pull request by CODeRUS**
@SfietKonstantin

### CODE REVIEW

File: rpm/patchmanager.spec [hunk 1]
```diff
@@ -61,6 +61,17 @@ on your SailfishOS device easily. This package
 contains the Jolla settings plugin.
 
 
+%package -n sailfishos-%{name}-unapplyall
```
Comment (SfietKonstantin) [inline rpm/patchmanager.spec]: Make it part of patchmanager. Do not create a new package

File: src/unapply/unified_diff.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+         contentHeight: content.height
+@@ -250,10 +267,7 @@ Page {
+                             }
+                         }
+                     } else if (storeIf.mayInstall) {
+-                        // Let the UI smoothly fade out first before doing anything scary
+-                        shutdownBus.call("setShutdownMode", ["upgrade"])
+-                        // Clear the cache and the system update notification
+-                        storeIf.clearOsUpdateCheckCache()
++                        patchmanagerBus.call("unapplyAllPatches", [], unapplyAllCallback)
```
Comment (SfietKonstantin) [inline src/unapply/unified_diff.patch]: Maybe there could be a problem here: what do we do when we failed to unapply patches ?

Comment (CODeRUS) [inline src/unapply/unified_diff.patch]: we can't upgrade sfos until some patch failed to unapply, that's the point.

File: rpm/patchmanager.spec (hunk 1 above)
Comment (CODeRUS) [inline rpm/patchmanager.spec]: hm, okay

File: src/unapply/unified_diff.patch (hunk 2 above)
Comment (SfietKonstantin) [inline src/unapply/unified_diff.patch]: My fault, I didn't reviewed it correctly ...

**Comment by CODeRUS** [comment]
done fixing comments :)

**Pull request merged**